gensim/summarization/bm25.py

```diff
@@ -225,7 +225,25 @@ def _get_scores(bm25, document):
     return bm25.get_scores(document)
 
 
-def iter_bm25_bow(corpus, n_jobs=1):
+def _bow_to_text(corpus):
+    """Helper function for transforming corpus from BOW-format to list of words format.
+
+    Parameters
+    ----------
+    corpus : corpus in BOW-format (word, weight).
+
+    Returns
+    -------
+    list of documents (list of words).
+    """
+    return [
+        [word
+            for word, count in doc
+            for _ in range(count)]
+        for doc in corpus]
+
+
+def iter_bm25_bow(corpus, n_jobs=1, is_bow=False):
```

Comment:
Please update the documentation to include the new parameter.

Also, please add unit tests for the new functionality.